src/PayID/pay-id-utils.ts

```diff
@@ -7,20 +7,26 @@ export default class PayIDUtils {
   /**
    * Parse a PayID string to a set of PayIDComponents object
    *
-   * @param parsePayID The input Pay ID.
+   * @param parsePayID - The input Pay ID.
    * @returns A PayIDComponents object if the input was valid, otherwise undefined.
    */
   public static parsePayID(payID: string): PayIDComponents | undefined {
-    if (!PayIDUtils.isASCII(payID)) return
+    if (!PayIDUtils.isASCII(payID)) {
+      return
+    }
```

Comment:
I wrote a rule banning single line `if` statements.

I go back and forth, but in general, I think it's reasonable to be consistent, even if this is a little more verbose in terms of LoC.